### PR TITLE
chore: fix typing for untyped defs

### PR DESCRIPTION
**Pull request by dodamih**

### CODE REVIEW

**Comment by dodamih** [comment]
Most of the ``type: ignore`` that are left are ``torchfields`` and ``docker`` not being typed.

File: zetta_utils/tensor_ops/mask.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 class TensorOp(Protocol[P]):
     """
-    Protocol which defines what it means for a funciton to be a tensor_op:
+    Protocol which defines what it means for a functoon to be a tensor_op:
```
[the remark this answers is not in the document]
Comment (nkemnitz) [reply zetta_utils/tensor_ops/mask.py:25]: funkytown!

File: zetta_utils/log.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 """Output logging."""
 # pylint: disable=global-statement
+# mypy: disable-error-code="unreachable"
```
[the remark this answers is not in the document]
Comment (dodamih) [reply zetta_utils/log.py:3]: The part where LOKI_HANDLER gets changed by outside code.

File: zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py [hunk 3]
```diff
@@ -303,7 +303,7 @@ def __attrs_post_init__(self):
                 raise ValueError(
                     f" `processing_blend_pad` must be less than or equal to"
                     f" half of `processing_chunk_size`; received {self.processing_blend_pad}",
-                    f" which is larger than {self.processing_size // 2}",
+                    f" which is larger than {self.processing_chunk_size // 2}",
```
[the remark this answers is not in the document]
Comment (dodamih) [reply zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py:306]: Yeah, should have been caught earlier...

File: zetta_utils/training/lightning/regimes/alignment/encoding_coarsener_highres.py [hunk 4]
```diff
@@ -83,10 +85,10 @@ def __attrs_pre_init__(self):
 
     def __attrs_post_init__(self):
         if self.encoder_ckpt_path is not None:
-            convnet.utils.load_model(self, self.encoder_ckpt_path, ["encoder"])
+            convnet.utils.load_weights_file(self, self.encoder_ckpt_path, ["encoder"])
```
[the remark this answers is not in the document]
Comment (dodamih) [reply zetta_utils/training/lightning/regimes/alignment/encoding_coarsener_highres.py:88]: Yeah, looks like it.

File: zetta_utils/log.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dodamih) [reply zetta_utils/log.py:3]: mypy doesn't seem to track module level globals, but that kinda makes sense for a static analyser.

[the remark this answers is not in the document]
Comment (supersergiy) [reply zetta_utils/log.py:3]: We should just annotate the variable properly

**Review by supersergiy — APPROVED** [review]
Looks great!

**Pull request merged**